index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="/styles.css" />
+  </head>
+  <body>
+    <div class="wrapper">
+      <header class="header-box">
+        <p>John Doe</p>
+        <img src="/images/Menu-icon.png" alt="" />
+      </header>
+        <section class="name-info">
+          <h1>Ukwishaka Levy</h1>
+          <h2>Software Developer</h2>
+        </section>
+        <section class="box-tools">
+          <p id="intro">
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hesitate to contact me.
+          </p>
+          <img src="/images/vector-line.png" alt="line" />
+          <div class="skills-box">
+            <div>
+              <h4>Languages</h4>
+              <p>Javascript</p>
+              <p>Ruby</p>
+              <p>Html</p>
+              <p>Css</p>
+            </div>
+            <div>
+              <h4>Frameworks</h4>
+              <p>Bootstrap</p>
+              <p>Ruby on Rails</p>
+              <p>RSPec</p>
+              <p>CapyBara</p>
+              <p>Selenium</p>
+            </div>
+            <div>
+              <h4>Skills</h4>
+              <p>Codekit</p>
+              <p>Github</p>
+              <p>Codepen</p>
+              <p>Gitlab</p>
+              <p>Terminal</p>
+            </div>
+          </div>
```

Comment:
- [ ] Please consider using a `ul` tag to wrap these skills and replacing the `divs` inside it with a `li` tag as the skills are an unordered list. This will improve the semantic structure of your HTML. See my suggestion below:

```suggestion
          <div class="skills-box">
            <ul>
              <li>
                Languages
                <ul>
                  <li>Javascript</li>
                  <li>Ruby</li>
                  <li>Html</li>
                  <li>Css</li>
                </ul>
               </li>
               <li>
                Frameworks
                <ul>
                  <li>Bootstrap</li>
                  <li>Ruby on Rails</li>
                  <li>RSPec</li>
                  <li>CapyBara</li>
                  <li>Selenium</li>
                </ul>
               </li>
               <li>
                Skills
                <ul>
                  <li>Skills</li>
                  <li>Codekit</li>
                  <li>Github</li>
                  <li>Codepen</li>
                  <li>Gitlab</li>
                  <li>Terminal</li>
                </ul>
               </li>
            </ul>
          </div>
```